src/components/ComponentPages/ProfileInfo/index.tsx

```diff
@@ -19,7 +19,19 @@ const ProfileInfo = () => {
 
   //on update profile click
   const onFinish = async (values: any) => {
-    values.private_flags = privateList.join();
+    //Set Private Public flags
```

Comment:
Can you create an associative array and then iterate?

```javascript
let flas = {
'first_name' : 'first_name_bit',
'last_name': 'last_name_bit'
//.....
};
```

Something like above.